lambdas/service/app.py

```diff
@@ -902,6 +902,10 @@ def filter_schema(relation: str) -> JSON:
 
 
 def repository_search_params_spec():
+    # Since this spec is used for all entity types and the schema validation
+    # isn't dynamic by which entity type is selected, we have to allow the
+    # largest possible value.
```

Comment:
Phrasing is a bit off. How about:

```suggestion
    # This spec is shared across all entity types, so no single value would be correct here.
    # Using the largest possible value ensures that we don't incorrectly reject valid requests, at the cost
    # of accepting some invalid ones.
```